R/lfd.r

```diff
@@ -1,10 +1,17 @@
 library("readODS")
+library("readxl")
```

Comment:
nly need due to a stray `read_excel` which has now been removed in master - this line can go now